types/react-bootstrap/index.d.ts

```diff
@@ -112,6 +112,28 @@ declare namespace ReactBootstrap {
     var ButtonGroup: React.ClassicComponentClass<ButtonGroupProps>;
 
 
+    // <ToggleButtonGroup />
+    interface ToggleButtonGroupProps extends React.HTMLProps<ToggleButtonGroup> {
+        /** Required if `type` is set to 'radio' */
+        name?: string;
+        type: 'radio' | 'checkbox';
+        defaultValue?: any;
+        value?: any;
```

Comment:
I think that's a good idea. From what I can tell, `any` is correct, but it would be very unusual to see anything but number or string here.